routes/api/user/update-user.js

```diff
@@ -7,5 +7,23 @@ var router = express.Router();
 
 console.log('update-user');
 
+app.get("/api/users/:id", function(req, res) {
+    // Add sequelize code to find a single post where the id is equal to req.params.id,
+    db.User.update({
+      user_name: req.body.user_name,
+      email: req.body.email,
+      about_me: req.body.about,
+    }, {
+      where: {
+        id: req.body.id
+      }
+    })
+      .then(function(dbUser) {
+        res.json(dbUser);
+      });
+    // return the result to the user with res.json
+  });
+
+
```

Comment:
Let's add a few `router.put` routes here:

1. Points to `/api/users/password` and calls `db.Users.update` on `password` where the `id` is equal to the one in the request body
2. Points to `/api/users/rating` and calls `db.Users.update` on `user_rating` where the `id` is equal to the one in the request body

We'll worry about Social Points in the MDP